web/libs/editor/src/components/BottomBar/buttons.tsx

```diff
@@ -50,22 +49,20 @@ type AcceptButtonProps = {
 export const AcceptButton = memo(
   observer(({ disabled, history, store }: AcceptButtonProps) => {
     return (
-      <ButtonTooltip key="accept" title="Accept annotation: [ Ctrl+Enter ]">
-        <Button
-          aria-label="accept-annotation"
-          disabled={disabled}
-          look="primary"
-          onClick={async () => {
-            const selected = store.annotationStore?.selected;
+      <Button
+        aria-label="accept-annotation"
```

Comment:
These buttons are used in a list, so should have a `key` here to not trigger react warnings
```suggestion
      <Button
        key="accept"
        aria-label="accept-annotation"
```